regression-tests/convert-FITS_no_arguments/convert-FITS_no_argument.sh

```diff
@@ -3,19 +3,28 @@
 die() { echo "FATAL ERROR IN CONVERT FITS: $@" >&2; exit 1;}
 TEST_RESULT=1
 
+abs(){
```

Comment:
Oh. Um. This is a Bourne shell function, and won't affect anything inside awk. Have you actually tried running this regression test?